httpd-client/tests/client.test.ts

```diff
@@ -0,0 +1,18 @@
+import { describe, expect, test } from "vitest";
+import { HttpdClient } from "../index";
+
+const client = new HttpdClient("http://0.0.0.0:8080");
```

Comment:
I find it odd that this IP works. Shouldn’t it be `http://127.0.0.1:8080`?

Reply:
Correct: https://github.com/radicle-dev/radicle-interface/pull/664.